libs/domains/database/src/lib/slices/databases.slice.ts

```diff
@@ -26,15 +43,71 @@ export const fetchDatabases = createAsyncThunk<Database[], { environmentId: stri
       thunkApi.dispatch(fetchDatabasesStatus({ environmentId: data.environmentId }))
     }
 
-    return response.data.results as Database[]
+    const databases = databaseFactoryMock(3)
+
+    /** @TODO replace with proper data */
+    const withMockedEnvId = databases.map((database) => {
```

Comment:
Why have you started by mocking the result here? Was it not quicker to directly plug it?